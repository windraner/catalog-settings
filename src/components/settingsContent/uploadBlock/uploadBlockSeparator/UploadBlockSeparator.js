import React from 'react';

import './uploadBlockSeparator.css';

 const UploadBlockSeparator = () => {
  return (
    <div className="catalog-settings-content__upload-separator">
      <div className="catalog-settings-content__upload-separator-border"></div>
      <div className="catalog-settings-content__upload-separator-text">OR</div>
      <div className="catalog-settings-content__upload-separator-border"></div>
    </div>
  );
};

export default UploadBlockSeparator;
