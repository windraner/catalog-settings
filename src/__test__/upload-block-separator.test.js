import React from 'react';
import UploadBlockSeparator from '../components/settings-content/upload-block/upload-block-separator/upload-block-separator.js';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('<UploadBlockSeparator />', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<UploadBlockSeparator />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
