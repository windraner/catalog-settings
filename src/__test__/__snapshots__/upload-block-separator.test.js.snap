// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UploadBlockSeparator /> matches the snapshot 1`] = `
<div
  className="catalog-settings-content__upload-separator"
>
  <div
    className="catalog-settings-content__upload-separator-border mx-auto my-0"
  />
  <div
    className="catalog-settings-content__upload-separator-text"
  >
    OR
  </div>
  <div
    className="catalog-settings-content__upload-separator-border mx-auto my-0"
  />
</div>
`;
